spec/requests/users_can_sign_out_spec.rb

```diff
@@ -5,10 +5,11 @@
     allow(ENV).to receive(:[]).with("BULLET_DEBUG").and_return("false")
     allow(ENV).to receive(:[]).with("AUTH0_CLIENT_ID").and_return("123456")
     allow(ENV).to receive(:[]).with("AUTH0_DOMAIN").and_return("test.auth0")
+    host! "test.local"
```

Comment:
If we have the global `config.before` do we still need this?

Reply:
I put this in so that the change on line 13 of `www.example.com` to `test.local` has some context.

Reply:
That is fair enough!